modules/javafx.graphics/src/main/java/com/sun/prism/impl/GlyphCache.java

```diff
@@ -239,6 +239,8 @@ private void clearAll() {
     }
 
     private GlyphData getCachedGlyph(int glyphCode, int subPixel) {
+        if (glyphCode < 0) {return null;}
```

Comment:
Have you ensured that all callers are prepared to deal with a `null` return value?

Minor: Please format this according to code conventions:

```
        if (glyphCode < 0) {
            return null;
        }
```

Reply:
> Have you ensured that all callers are prepared to deal with a `null` return value?
> 
> Minor: Please format this according to code conventions:
> 
> ```
>         if (glyphCode < 0) {
>             return null;
>         }
> ```



> Have you ensured that all callers are prepared to deal with a `null` return value?
> 
> Minor: Please format this according to code conventions:
> 
> ```
>         if (glyphCode < 0) {
>             return null;
>         }
> ```

`      if (!packer.add(rect)) {
                    if (PULSE_LOGGING_ENABLED) {
                        PulseLogger.incrementCounter("Font Glyph Cache Cleared");
                    }
                    // If add fails,clear up the cache. Try add again.
                    clearAll();
                    if (!packer.add(rect)) {
                        if (PrismSettings.verbose) {
                            System.out.println(rect + " won't fit in GlyphCache");
                        }
                        return null;
                    }
                }`


The line also return null,so return nul is ok